test/integration/model/create.test.js

```diff
@@ -894,7 +894,7 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         });
       });
     });
-    it('raises an error if created object breaks definition contraints', function() {
+    it.only('raises an error if created object breaks definition contraints', function() {
```

Comment:
oh! oh!!!! was always wondering if there is a way of only executing specific tests in mocha :)